Procfile

```diff
@@ -5,6 +5,7 @@
 # Procfile for development using the new threaded worker (scheduler, twitter stream and delayed job)
 web: bundle exec rails server -p ${PORT-3000} -b ${IP-0.0.0.0}
 jobs: bundle exec rails runner bin/threaded.rb
+dj: bundle exec script/delayed_job run
```

Comment:
Not needed. I was having some trouble with things running in a timely manner in my local env so added this and forgot to take out.